packages/components/src/sites-table/sites-table-filter-tabs.tsx

```diff
@@ -0,0 +1,50 @@
+import { TabPanel } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import type { SiteData } from 'calypso/state/ui/selectors/site-data'; // eslint-disable-line no-restricted-imports
+
+interface SitesTableFilterTabsProps {
+	allSites: SiteData[];
+	children( filteredSites: SiteData[] ): JSX.Element;
+	className?: string;
+}
+
+export function SitesTableFilterTabs( {
+	allSites,
+	children: renderContents,
+	className,
+}: SitesTableFilterTabsProps ) {
+	const { __ } = useI18n();
+
+	return (
+		<TabPanel
+			className={ className }
+			tabs={ [
+				{ name: 'all', title: __( 'All', __i18n_text_domain__ ) },
```

Comment:
Will we need to add the `__i18n_text_domain__` for _all_ instances of `__()`? If yes, how about we create a proxy hook that injects this second argument for us?

Reply:
It's only necessary for instances that are inside the `packages/` folder. Anything inside `client/sites-dashboard/` folder don't need it.

Unfortunately it's not possible to write a wrapper. It's the same reason you can't pass a string variable to the `__()` function: our translation tools statically analyse the code looking for those `__` functions, so they have to be in the form that the tool is able to understand.
I suppose it is possible that we modify our translation tools to be more aware of packages. You can read more about possible solutions here: pbAok1-1Ao-p2